java/ql/src/Security/CWE/CWE-926/ImplicitlyExportedAndroidComponent.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @name Implicitly exported Android component
+ * @description An Android component with an '<intent-filter>' and no 'android:exported' attribute is implicitly exported. This can allow for improper access to the component and its data.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 8.2
+ * @id java/android/implicitly-exported-component
+ * @tags security
+ *       external/cwe/cwe-926
+ * @precision medium
```

Comment:
I know I'm the one who suggested this to be `medium`, but I'm re-considering it. After all, the query _is_ precise: most of the time (if not always), it will correctly flag an implicitly exported component.

What we don't know for sure is whether it's intentionally exported or not, and thus whether the issue has security implications, but that is better conveyed by `@problem.severity warning` I think. So IMHO `precision` could be raised to `high`.

Reply:
Updated in commit [bb1b135](https://github.com/github/codeql/pull/9983/commits/bb1b135380f9fe60c13d6e158bb20912a1eb2d28).